pkg/universe.dagger.io/x/paul@fgtech.fr/ansible/test/test.cue

```diff
@@ -0,0 +1,60 @@
+package test
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+	"universe.dagger.io/x/paul@fgtech.fr/ansible"
+)
+
+dagger.#Plan & {
+
+	client: filesystem: ".": read: contents:         dagger.#FS
+	client: filesystem: "./output": write: contents: actions.playbook.output
+
+	actions: {
+		load: {
+			sources:     client.filesystem.".".read.contents
+			_loadConfig: core.#Source & {
+				path: "./config"
+				include: ["*.cfg"]
+			}
+			_loadInventory: core.#Source & {
+				path: "./inventory"
+				include: ["*"]
+			}
+			_loadPlaybook: core.#Source & {
+				path: "./project"
+				include: ["*"]
+			}
+			_loadSSH: core.#Source & {
+				path: "./ssh"
+				include: ["*"]
+			}
```

Comment:
You could also read it with client API instead of sourcing. I'm not sure what is the best actually